src/components/shared/transactionPriority/transactionPriority.js

```diff
@@ -132,6 +136,7 @@ const TransactionPriority = ({
               className={`${styles.priorityTitle} ${index === selectedPriority ? styles.priorityTitleSelected : ''} option-${priority.title}`}
               onClick={onClickPriority}
               value={index}
+              disabled={index === 0 ? false : shouldDisableOptions}
```

Comment:
I'd put `disabled={index && !priority.value}` and get rid if `shouldDisableOptions `.